frontend/src/app/components/modals/create-namespace-bucket-modal/create-namespace-bucket-modal.js

```diff
@@ -169,7 +172,8 @@ class CreateNamespaceBucketModalViewModel extends ConnectableViewModel {
             isStepValid: isFormValid(form),
             readPolicy,
             writePolicy,
-            cacheTTL
+            cacheTTL,
+            cacheEvictionType
```

Comment:
There is no `cacheEvictionType` prop on the view model (the current class), so this assignment is meaningless.
Also, I am not sure if you need it on the view model at all, because in the view you could always read it from `$form.cacheEvictionType`

Reply:
Removed